apps/passport-client/components/screens/ProveScreen/SemaphoreSignatureProveScreen.tsx

```diff
@@ -47,12 +47,12 @@ export function SemaphoreSignatureProveScreen({
         const pendingPCD = await requestPendingPCD(serverReq);
         window.location.href = `${
           req.returnUrl
-        }?encodedPendingPCD=${JSON.stringify(pendingPCD)}`;
+        }?srcId=${req.srcId}&encodedPendingPCD=${JSON.stringify(pendingPCD)}`;
```

Comment:
might be nice to extract this sort of url construction into `passport-interface` package as well at some point